app.js

```diff
@@ -0,0 +1,50 @@
+const glob = require("glob");

```

Comment:
try to have all imports at top of the file

Reply:
fixed this